vendor/engines/bulk_email/app/helpers/bulk_email/application_helper.rb

```diff
@@ -0,0 +1,18 @@
+module BulkEmail
+
+  module ApplicationHelper
+
+    include Rails.application.routes.url_helpers
+
+    def admin_instrument_send_mail_path(instrument)
+      facility_bulk_email_path(
+        products: [instrument.id],
+        start_date: l(Date.today, format: :usa),
```

Comment:
If having no user type selected returns nothing, then we should probably default it to "Customers". That way you can come through and all you have to click is "Search".